spec/unit/provider/package/zypper_spec.rb

```diff
@@ -136,7 +127,7 @@ def shell_out_expectation!(command, options=nil)
       shell_out_expectation!(
         "zypper --non-interactive install --auto-agree-with-licenses emacs=1.0"
```

Comment:
:+1: 

(and I would agree with you on that argument... it wasn't at all obvious to me that it should upgrade when running `zypper install`... That's why I'm proposing we add a line comment there to document this non-obvious behaviour)